cypress/e2e/utils/env.ts

```diff
@@ -0,0 +1,111 @@
+export const loadPlayer = (pluginConf = {}, playbackConf: Record<string, any> = {}) => {
+  cy.visit('index.html');
+  return cy.window().then(win => {
+    try {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const kalturaPlayer = win.KalturaPlayer.setup({
+        targetId: 'player-placeholder',
+        provider: {
+          partnerId: -1,
+          // ks: '5678',
+          env: {
+            cdnUrl: 'http://mock-cdn',
+            serviceUrl: 'http://mock-api'
+          }
+        },
+        sources: {
+          metadata: {
+            name: 'download'
+          }
+        },
+        plugins: {
+          kava: {
+            disable: true
+          },
+          download: pluginConf,
+          uiManagers: {}
+        },
+        ui: {
+          translations: {
+            en: {
+              download: {
+                downloads: 'Downloads',
```

Comment:
why set translations ? I would prefer to not set what isn't relevant

Reply:
if i don't set it then i don't see the header in the layout